src/main/java/com/example/solidconnection/siteuser/controller/MyPageController.java

```diff
@@ -49,4 +50,13 @@ public ResponseEntity<Void> updatePassword(
         myPageService.updatePassword(siteUserId, request);
         return ResponseEntity.ok().build();
     }
+
+    @PatchMapping("/location")
+    public ResponseEntity<Void> updateLocation(
+            @AuthorizedUser long siteUserId,
+            @RequestBody LocationUpdateRequest request
```

Comment:
```suggestion
            @RequestBody @Valid LocationUpdateRequest request
```

`@Valid` 어노테이션이 있어야 Dto 어노테이션 검증이 가능합니다!

Reply:
헛 ... 이걸 빼먹다니 ... 수정했습니다

Reply:
https://github.com/solid-connection/solid-connect-server/pull/453/commits/5ed701f783ca6624126634bbed714ebefbfd0111